packages/@local/eslint-config/index.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
     "no-nested-ternary": "off",
     "no-restricted-syntax": "off",
     camelcase: "off",
+    curly: ["error", "all"],
```

Comment:
👀

Reply:
`all` is default, but the rule works as `multi` if the argument is not passed. This could be to do with rule inheritance: AirBnB config must be setting `multi` for us.